### PR TITLE
[tvm] numpy op fmax

### PR DESCRIPTION
**Pull request by hgt312**
## Description ##
Numpy op `fmax` implementation using tvm.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- add `fmax`

## Comments ##
Thanks for reviewing.

### CODE REVIEW

File: src/operator/contrib/tvmop/ufunc.cc [hunk 1]
```diff
@@ -61,6 +67,30 @@ NNVM_REGISTER_OP(_contrib_tvm_vadd)
...
+    .add_argument("a", "NDArray-or-Symbol", "first input")
+    .add_argument("b", "NDArray-or-Symbol", "second input")
+    .set_attr<mxnet::FInferShape>("FInferShape", BinaryBroadcastShape)
+    .set_attr<nnvm::FInferType>("FInferType", mxnet::op::ElemwiseType<2, 1>)
+#if MXNET_USE_CUDA
+    .set_attr<mxnet::FCompute>("FCompute<gpu>", mxnet::op::TVMBroadcastCompute<func_fmax_gpu>)
+#endif  // MXNET_USE_CUDA
+    .set_attr<mxnet::FCompute>("FCompute<cpu>", mxnet::op::TVMBroadcastCompute<func_fmax_cpu>);
+
+NNVM_REGISTER_OP(_contrib_tvm_fmin)
```
Comment (yzhliu) [inline src/operator/contrib/tvmop/ufunc.cc]: let's put it in regular numpy namespace

File: src/operator/contrib/tvmop/ufunc.cc [hunk 2]
```diff
@@ -61,6 +67,30 @@ NNVM_REGISTER_OP(_contrib_tvm_vadd)
...
+    .set_num_inputs(2)
+    .set_num_outputs(1)
+    .add_argument("a", "NDArray-or-Symbol", "first input")
+    .add_argument("b", "NDArray-or-Symbol", "second input")
+    .set_attr<mxnet::FInferShape>("FInferShape", BinaryBroadcastShape)
+    .set_attr<nnvm::FInferType>("FInferType", mxnet::op::ElemwiseType<2, 1>)
+#if MXNET_USE_CUDA
+    .set_attr<mxnet::FCompute>("FCompute<gpu>", mxnet::op::TVMBroadcastCompute<func_fmin_gpu>)
```
Comment (yzhliu) [inline src/operator/contrib/tvmop/ufunc.cc]: maybe rename it as `TVMBinaryCompute` instead.

Comment (yzhliu) [reply src/operator/contrib/tvmop/ufunc.cc]: also add gradient.

**Comment by hgt312** [comment]
split to two pr, closed

**Pull request closed without merging**